runtime/tests/port/j9vmemTest.c

```diff
@@ -2582,6 +2753,8 @@ j9vmem_testFindValidPageSize(struct J9PortLibrary *portLibrary)
 	memcpy(old_vmem_pageSize, pageSizes, J9PORT_VMEM_PAGESIZE_COUNT * sizeof(UDATA));
 	memcpy(old_vmem_pageFlags, pageFlags, J9PORT_VMEM_PAGESIZE_COUNT * sizeof(UDATA));
 
+	/* for AIX, the default size is 4K, the possible large page sizes are 4K, 64K, 16M, 16G */
```

Comment:
Each sentence should start with a capital and end with a period. Again, suggest "on" is more appropriate than "for".
```c
	/* On AIX, the default size is 4K, the possible large page sizes are 4K, 64K, 16M, 16G. */
```